src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/RazorCodeActionsTests.cs

```diff
@@ -26,7 +27,11 @@ public async Task RazorCodeActions_AddUsing()
         var codeActions = await TestServices.Editor.InvokeCodeActionListAsync(ControlledHangMitigatingCancellationToken);
 
         // Assert
-        var codeActionSet = Assert.Single(codeActions);
+
+        // We expect two groups, one for Razor, one for Html
+        Assert.Equal(2, codeActions.Count());
```

Comment:
Still don't fully understand. Do we have an empty group?

Reply:
No, we have two groups, each with one code action. Add using, and Remove tag (from html). They used to be in one group

Reply:
Ohhhh, I see. Line 36 is `Assert.Single` not `Assert.Collection` so it wouldn't fail prior because it was finding one "Add using" code action. 🤦

Reply:
Yah, it confused me at first too. I thought somehow my change was causing multiple code actions to appear or something. My dislike tor `Single` only grows :)